aidants_connect_habilitation/models.py

```diff
@@ -1 +1,145 @@
-# Create your models here.
+from django.db import models
+from django.db.models import Q
+from phonenumber_field.modelfields import PhoneNumberField
+
+from aidants_connect_habilitation.constants import (
+    RequestStatusConstants,
+    RequestOriginConstants,
+)
+
+
+class Issuer(models.Model):
+    first_name = models.CharField("Prénom", max_length=150)
+    last_name = models.CharField("Nom", max_length=150)
+    email = models.EmailField(max_length=150)
+    profession = models.CharField("Intitulé du poste", blank=False, max_length=150)
+    phone = PhoneNumberField(blank=True)
+
+
+class OrganisationRequest(models.Model):
+    issuer = models.ForeignKey(
+        Issuer,
+        null=True,
+        on_delete=models.CASCADE,
+        related_name="organisation_requests",
+    )
+
+    status = models.CharField(
+        "État",
+        blank=False,
+        max_length=150,
+        default=RequestStatusConstants.NEW.name,
+        choices=RequestStatusConstants.choices(),
+    )
+
+    type = models.CharField(
+        "Type de structure",
+        blank=False,
+        max_length=200,
+        choices=RequestOriginConstants.choices(),
+    )
+
+    type_other = models.CharField(
+        "Type de structure",
+        blank=True,
+        null=True,
+        default=None,
+        max_length=200,
+    )
+
+    # Organisation
+    name = models.TextField("Nom", default="No name provided")
+    siret = models.BigIntegerField("N° SIRET", default=1)
+    address = models.TextField("Adresse", default="No address provided")
+    zipcode = models.CharField("Code Postal", max_length=10, default="0")
+    city = models.CharField("Ville", max_length=255, null=True)
+
+    partner_administration = models.CharField(
+        "Administration partenaire",
+        blank=True,
+        null=True,
+        default=None,
+        max_length=200,
+    )
+
+    public_service_delegation_attestation = models.FileField(
+        "Attestation de délégation de service public"
+    )
+
+    france_service_label = models.BooleanField(
+        "Labellisation France Services", default=False
+    )
+
+    web_site = models.URLField("Site web")
+
+    # max_lengh?
+    mission_description = models.TextField("Description des missions de la structure")
+
+    avg_nb_demarches = models.IntegerField(
+        "Nombre moyen de démarches ou de dossiers traités par semaine"
+    )
+
+    # Manager
+    manager_first_name = models.CharField("Prénom", max_length=150)
+    manager_last_name = models.CharField("Nom", max_length=150)
+    manager_email = models.EmailField(max_length=150)
+    manager_profession = models.CharField(
+        "Intitulé du poste", blank=False, max_length=150
+    )
+
+    # Checkboxes
+    cgu = models.BooleanField("J'accepte les CGU")
+    dpo = models.BooleanField("Le DPO est informé")
+    aidants_only = models.BooleanField(
```

Comment:
Je suggère plutôt : 

```suggestion
    professionals_only = models.BooleanField(
```

J'ai eu du mal à comprendre de quoi on parlait quand j'ai vu ce champ dans la liste des contraintes, donc sans son libellé.